docs/contributing/contributing-toolbox.rst

```diff
@@ -0,0 +1,53 @@
+.. _sec-contributing-toolbox:
+
+Toolbox
+=======
+
+Supporting the developtment of **xNVMe** is a toolbox with a collection of
+scripts in Python, Bash and a ``Makefile`` to instrument it. See section
+:ref:`sec-contributing-toolbox-mkhelper` for a description of it.
+
+For common linter/format tooling then the `pre-commit`_ framework is used and
+described in section :ref:`sec-contributing-toolbox-precommit`.
+
+.. _sec-contributing-toolbox-mkhelper:
+
+Makefile helper
+---------------
+
+As described in :ref:`sec-getting_started`, then **xNVMe** uses the `Meson`_
```

Comment:
Removing "then" improves the readability

Reply:
Changed, thanks!